BraintreeCore/src/main/java/com/braintreepayments/api/GooglePayConfiguration.kt

```diff
@@ -22,7 +22,7 @@ internal data class GooglePayConfiguration constructor(
     val paypalClientId: String
 ) {
 
-    constructor(json: JSONObject?) : this(
+    internal constructor(json: JSONObject?) : this(
```

Comment:
I'm not sure if this needs to be made `internal` since the class is already marked `internal`.